build/docs/CHANGELOG.md

```diff
@@ -14,7 +14,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [0.0.0.29] - 2023-03-30
 
-- `cluster-manager` now waits for `v1-validator001` to be online AND responsive by checking the `/v1/health` endpoint (dogfooding)
+- `cluster-manager` now waits for `validator-001-pocket-validator` to be online AND responsive by checking the `/v1/health` endpoint (dogfooding)
```

Comment:
Not confirmed yet but this rename to `validator-001-pocket-validator` could break E2E tests. It would be a small fix but just wanted to make you aware.

Reply:
@dylanlott makes sense. Thanks for the heads-up! We are going to need to adjust the name, is it configurable by the way, via environment variable or otherwise?

Reply:
It's just a hardcoded variable right now, but environment config isn't a bad idea. I will see what that looks like. I will have to write documentation for how to change the name of the validator that the tests use in E2E scenarios.

Reply:
@dylanlott I see, this code now is in `main` branch - I'll change the template for now, and we can take a look at making it configurable later.